docs/guides/src/main/server/proxy.adoc

```diff
@@ -18,22 +19,22 @@ The `reencrypt` mode requires communication through HTTPS between the proxy and
 
 The `passthrough` mode enables communication through HTTP or HTTPS between the proxy and Keycloak. This mode is suitable for deployments where the reverse proxy is not terminating TLS, but only forwarding the requests to the Keycloak server so that secure connections between the server and clients are based on the keys and certificates used by the Keycloak server itself.
 
-Selecting the reverse proxy mode is done at runtime via:
+== Configure the proxy mode in Keycloak
+To select the proxy mode, run:
 
 <@kc.all parameters="--proxy <mode>"/>
```

Comment:
I'm not super keen on @kc.all as that could be used by all "runtime" options, and this can easily lead into a bit of a mix/match. I'd rather just stick with a general rule of using `kc build` for build-time options, and `kc start` for runtime options. Then in how to configure guide we can cover that runtime options can also be included in config files, added to build command, etc.

Reply:
@kc.all would also be pretty hard to use if we decide to do the whole different versions of the docs for different environments.